sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosResponseDiagnostics.java

```diff
@@ -16,6 +17,11 @@
 public class CosmosResponseDiagnostics {
     private static final Logger logger = LoggerFactory.getLogger(CosmosResponseDiagnostics.class);
     private static final ObjectMapper objectMapper = new ObjectMapper();
+
```

Comment:
Though its not related to your PR, but I am wondering, why do we have separate `ObjectMapper` instance here ?
Why don't we use the one from Utils.java ?
Reason for asking this question is, we don't register `objectMapper` with all the modules here.

Reply:
The ObjectMapper from Utils is used for handling user data, and may require special configuration.
This one is used for handling internal constructs not user data.